File: Service_eHospital_DongNai_A/Implement/CSKH_DM_GoiDichVuService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class CSKH_DM_GoiDichVuService : BaseService<CSKH_DM_GoiDichVu, ICSKH_DM_GoiDichVuRepository>
    , ICSKH_DM_GoiDichVuService
    {
    private readonly ICSKH_DM_GoiDichVuRepository _CSKH_DM_GoiDichVuRepository;
    public CSKH_DM_GoiDichVuService(ICSKH_DM_GoiDichVuRepository CSKH_DM_GoiDichVuRepository) : base(CSKH_DM_GoiDichVuRepository)
    {
    _CSKH_DM_GoiDichVuRepository = CSKH_DM_GoiDichVuRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/CSKH_DM_GoiDichVu.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CSKH_DM_GoiDichVu } from './CSKH_DM_GoiDichVu.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CSKH_DM_GoiDichVuService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'GoiDichVu_Id', 'MaGoiDichVu', 'TenGoiDicVu', 'TenGoiDicVu_EN', 'TenGoiDicVu_RU', 'SoTienThucThu', 'SoTienGoiKham', 'TienTe_Id', 'TyGia', 'NgayHieuLuc', 'NgayHetHan', 'Idx', 'GhiChu', 'TamNgung', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id', 'Attribute1', 'Attribute2', 'Attribute3', 'Attribute4', 'Attribute5', 'GoiKhamSucKhoe', 'ChuyenKhoa_Id', 'Nam', 'Thang', 'Ngay', 'CostCenter_Id', 'SoNgayFU', 'KiemTraDieuKienKhoaGoi', 'CheckList'];

    List: CSKH_DM_GoiDichVu[] | undefined;
    ListFilter: CSKH_DM_GoiDichVu[] | undefined;
    FormData!: CSKH_DM_GoiDichVu;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CSKH_DM_GoiDichVu";
    }
}

